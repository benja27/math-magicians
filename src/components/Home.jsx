import React from 'react';

export default function Home() {
  return (
    <div className="px-5 d-flex flex-column pt-5">
      <h1>Welcome to our page</h1>
      <p className="fs-3">
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
      </p>
      <p className="fs-3">
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aperiam ea qui accusamus iure
      </p>
    </div>
  );
}
