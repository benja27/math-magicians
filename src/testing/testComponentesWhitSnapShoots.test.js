import {render, fireEvent, screen} from "@testing-library/react";
import Calculator from "../components/calculator";
import Home from "../components/Home";
import Navbar from "../components/Navbar";
import DisplayQuote from "../components/displayQuote";
import { BrowserRouter } from "react-router-dom";


test("test if elements inside calculator are render correctly", () => {
    render(<Calculator/>);
    const buttonAc = screen.getByText(/AC/i);
    expect(buttonAc).toBeInTheDocument();
});

test("test if calculator render correctly", () => {
    const {calculator} = render(<Calculator/>);
    expect(calculator).toMatchSnapshot();
});

test("test if Home renders correctly", () => {
    const {home} = render(<Home/>);
    expect(home).toMatchSnapshot();
});
test("test if Navbar render correctly", () => {
    const {navbar} = {
        render: () => (
            <BrowserRouter>
                <Navbar/>
            </BrowserRouter>
        )
    }
    expect(navbar).toMatchSnapshot();
});

test("test if Quote render correctly", () => {
    const {displayQuote} = render(<DisplayQuote/>);
    expect(displayQuote).toMatchSnapshot();
});