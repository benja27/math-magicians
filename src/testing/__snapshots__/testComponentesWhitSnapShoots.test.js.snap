// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if Home renders correctly 1`] = `undefined`;

exports[`test if Navbar render correctly 1`] = `undefined`;

exports[`test if Quote render correctly 1`] = `undefined`;

exports[`test if calculator render correctly 1`] = `undefined`;
